charts/index.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+entries:
+  meshery:
+  - apiVersion: v2
+    appVersion: latest
+    created: "2020-01-31T23:28:58.095925+08:00"
+    dependencies:
+    - condition: mesheryistio.enabled
+      name: mesheryistio
+      repository: ""
+      version: 0.1.0
+    - condition: mesherylinkerd.enabled
+      name: mesherylinkerd
+      repository: ""
+      version: 0.1.0
+    description: Meshery chart for deploying Meshery and Meshery's adapters.
+    digest: 65fa90d72355425b85c0299a711b652afb6e7f8f8b856b3c939883f049f60740
+    icon: https://github.com/layer5io/meshery/blob/master/ui/static/img/meshery-logo/meshery-logo.svg
+    keywords:
+    - meshery
+    - mesheryistio
+    - mesherylinkerd
+    maintainers:
+    - email: urakiny@gmail.com
+      name: aisuko
+    name: meshery
+    type: application
+    urls:
+    - meshery-0.1.0.tgz
```

Comment:
Given that the source for these charts in is the Meshery repo, instead of storing the compressed version of the charts in this repo as a charts "release", can we consider incorporating the tar.gz creation of these charts to our GitHub Actions workflow in Meshery? 

We would include the tar.gz creation part of the tagged workflow (the workflow that is enacted when a release tag is present). We could either have goreleaser tar.gz the charts or have a script executed to tar.gz the charts and make that tar.gz available alongside the rest of the Meshery release artifacts here - https://github.com/layer5io/meshery/releases

In this way, the Meshery helm chart is treated as another release artifact that is created each time we cut a new Meshery release.

Reply:
Good idea, maybe we can do this. The Meshery image's version was used `latest` under the chart. This tag which you said is the chart itself version. The Meshery version in the chart is here below.
 https://github.com/layer5io/meshery/blob/8596379830f5826379923afb4e24890b655f52b2/install/kubernetes/helm/meshery/Chart.yaml#L34